221/src/Tester.java

```diff
@@ -1,34 +1,95 @@
 
-    public static int fun100(int x, int y) {
-        if(x == 0)
-            return y;
-        else {
-            return fun100(x-1,x+y);
-        }
+    // Method used for adding each individual number together
+    public int addingUp(int topNum, int botNum) {
+        int value1 = topNum - '0';
+        int value2 = botNum - '0';
+        int addedSum = value1 + value2;
+        return addedSum;
     }
 
-    public static void main(String [] args) {
-    //int[] arr = {1, 2, 3, 4, 1};
-    //System.out.println(fun100(5, 10));
-//        Queue<Integer> q = new Queue<>();
-//        int item1 = 1;
-//        int item2 = 0;
-//        int item3 = 4;
-//        q.enqueue(item2);
-//        q.enqueue(item1);
-//        q.enqueue(item1 + item3);
-//        item2 = q.peek();
-//        q.enqueue(item3*item3);
-//        q.enqueue(item2);
-//        q.enqueue(3);
-//        item1 = q.peek();
-//        q.dequeue();
-//        System.out.println(item1 + " " + item2 + " " + item3);
-//        while (!q.isEmpty()) {
-//            System.out.println(q.dequeue());
-//        }
+    public static void main(String[] args) {
+
+        String total = "";
+        int carryOver = 0;
+        int placeHolder = 1;
+
+        Tester columnTotal = new Tester();
+        Scanner request = new Scanner(System.in);
+
+        // Requesting input, of two string as numbers, from the user
+        System.out.println("Type your first number: ");
+        String firstInput = request.next();
+        System.out.println("Type your second number: ");
+        String secondInput = request.next();
+
+        request.close();
+
+        int strOneLength = firstInput.length();
+        int strTwoLength = secondInput.length();
+
+        // 2 for loops checking for letters
+        for (int i = 0; i < strOneLength; i++) {
+            if (!Character.isDigit(firstInput.charAt(i)))  {
+                System.out.println("That is not a number.");
+                return;
+            }
+        }
+        for (int i = 0; i < strTwoLength; i++) {
+            if (!Character.isDigit(secondInput.charAt(i)))  {
+                System.out.println("That is not a number.");
+                return;
+            }
+        }
+        // 2 if statements checking for the length of the string from the user
+        if (strOneLength > strTwoLength) {
+            // For loops used to append a '0' at the front of the string
+            // if one string is longer than the other
+            for (int k = strTwoLength; strOneLength > k; k++) {
+                secondInput = "0" + secondInput;
+                strTwoLength = secondInput.length();
+            }
+        }
+        if (strTwoLength > strOneLength) {
+            for (int j = strOneLength; strTwoLength > j; j++) {
+                firstInput = "0" + firstInput;
+                strOneLength = firstInput.length();
+            }
+        }
+        System.out.println(firstInput + "  a " + secondInput); // Reads correctly---
```

Comment:
Move comment to separate line

Reply:
Next line has a comment, would you like me to make a new line?

Reply:
Yes please

Reply:
Moved comment to line above